src/app/public/src/modules/restricted-view/restricted-view.module.spec.ts

```diff
@@ -0,0 +1,14 @@
+import {
```

Comment:
@Blackbaud-BobbyEarl @Blackbaud-SteveBrush  Are spec.ts files expected and necessary for module exports at for coverage still?

Reply:
Nope! As long as you're importing the module into the component unit test, should be enough.